src/Ombi/ClientApp/src/app/requests-list/components/movies-grid/movies-grid.component.ts

```diff
@@ -46,10 +47,13 @@ export class MoviesGridComponent implements OnInit, AfterViewInit {
     @ViewChild(MatSort) sort: MatSort;
 
     constructor(private requestService: RequestServiceV2, private ref: ChangeDetectorRef,
-                private auth: AuthService, private storageService: StorageService,
-                private requestServiceV1: RequestService, private notification: NotificationService,
-                private translateService: TranslateService) {
+        private auth: AuthService, private storageService: StorageService,
+        private requestServiceV1: RequestService, private notification: NotificationService,
+        private translateService: TranslateService, private identity: IdentityService) {
 
+        identity.getUser().subscribe(u => {
```

Comment:
I think using the `auth.service.ts` and getting the Username from the claim would be better than calling an API?